packages/rxvt-unicode/README.md

```diff
@@ -0,0 +1,24 @@
+<p align="center">
+  <img src="https://github.com/daltonmenezes/assets/blob/master/images/aura-theme/heading.png?raw=true" alt="Aura Theme" width="70%" />
+</p>
+
+<p align="center">
+  <!-- Patreon -->
+  <a href="https://www.patreon.com/daltonmenezes">
+    <img alt="patreon url" src="https://img.shields.io/badge/support%20on-patreon-1C1E26?style=for-the-badge&labelColor=1C1E26&color=4dbb99">
+  </a>
+  <!-- version -->
+  <a href="#">
+    <img alt="version" src="https://img.shields.io/badge/version%20-v1.0.0-1C1E26?style=for-the-badge&labelColor=1C1E26&color=4dbb99">
+  </a>
+</p>
+
+> Aura Theme is a beautiful dark theme for rxvt terminal and another apps focused on long-term work without causing any discomfort to the eyes.
+
+# Installation
+1. Open [.Xresources](https://github.com/Douglaskav/aura-theme/blob/master/packages/rxvt-unicode/.Xresources) and save it in your home folder without txt extension that the browser will infer
```

Comment:
Please, update the repo username to **daltonmenezes**. Example:
From this:
```bash
https://github.com/Douglaskav/aura-theme/blob/master/packages/rxvt-unicode/.Xresources
```
to this:
```
https://github.com/daltonmenezes/aura-theme/blob/master/packages/rxvt-unicode/.Xresources
```